client/Components/stockCard.jsx

```diff
@@ -1,32 +1,85 @@
 'use strict';
-
+//NB.this page need to have the this.props.inputID to work
 import React from 'react';
 import { render } from 'react-dom';
 import { connect } from 'react-redux';
 import { toJS } from 'immutable';
 
+const Card = require('material-ui/lib/card/card');
+const CardActions = require('material-ui/lib/card/card-actions');
+const CardExpandable = require('material-ui/lib/card/card-expandable');
+const CardHeader = require('material-ui/lib/card/card-header');
+const CardMedia = require('material-ui/lib/card/card-media');
+const CardText = require('material-ui/lib/card/card-text');
+const CardTitle = require('material-ui/lib/card/card-title');
+const RaisedButton = require('material-ui/lib/raised-button');
+
 export const Stock = React.createClass({
 
+  //place to hold temporary variables
+  getInitialState() {
+      return {
+        numSharesToSell: 0
+      };
+  },
+
```

Comment:
Using getInitialState with redux seems dangerous, or at least like an antipattern. Is this necessary?